src/index.js

```diff
@@ -145,11 +145,8 @@ class DBLAPI extends EventEmitter {
       data.shard_count = shardCount;
     } else {
       data.server_count = this.client.guilds.size;
-      if (this.client.shard && this.client.shard.count) {
```

Comment:
You should do this without removing support for normal discord.js

Reply:
But normal d.js doesn't have client.ws, ofc you can check if(client.ws.shards), but I did it late night and to be honest I made it just to make it. Feel free to adjust it.

Reply:
It's quite literally as simple as bring back the code you removed and keep the new code